cmd/kubectl-gadget/utils/exec.go

```diff
@@ -62,7 +62,7 @@ func ExecPod(client *kubernetes.Clientset, node string, podCmd string, cmdStdout
 	}
 	podName := pods.Items[0].Name
 
-	restConfig, err := kubeRestConfig()
+	restConfig, err := kubeRestConfig(KubernetesConfigFlags)
```

Comment:
Does it make sense to pass that variable, perhaps kubeRestConfig() could take it directly? 

Also, I see it's the only place where `kubeRestConfig()` is used, other places use `utils.KubernetesConfigFlags.ToRESTConfig()` directly. Perhaps we can remove this function and use that directly as in other places?

Reply:
Thanks for the suggestion. Yeah, it makes sense to remove this function since it looks more clear e.g why we were doing `factory.SetKubernetesDefaults(restConfig)` etc. I updated the PR!